src/App.jsx

```diff
@@ -0,0 +1,20 @@
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import './App.css';
+import Header from './components/Header.jsx/Header';
+import Articles from './components/Articles';
+
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+function App() {
+	return (
+		<BrowserRouter>
+			<Header></Header>
+
+			<Routes>
+				<Route path="/" element={<Articles></Articles>}></Route>
+			</Routes>
+		</BrowserRouter>
+	);
```

Comment:
Nice routes setup. You can put your BrowserRouter tags around your App tag in main.jsx to clean things up if you'd like.